### DISCUSSION TITLE
Is there a chance to make the Notes app work with this module?

### DISCUSSION THREAD

**Discussion by LautiLosio** [reiryuki/Miui-Core-Magisk-Module#8] [Q&A] [answered]
Hi, I've been looking long for a good replacement for the native notes app, but I couldn't. This app has all the functionality that I need.

The issue is that I am now running AOSP and I'm not planning to go back to MIUI. But I'd love to have this app.

The closest I got was installing the magisk module, this solved the "couldn't insatll app" problem. However when I open the app it crashes and shows the following log: https://www.toptal.com/developers/hastebin/icusapejuw

I really hope this is possible.

Great work so far!

**Comment by reiryuki** [comment, marked as answer]
The notes app needs to be installed to /system

**Reply by LautiLosio** [reply]
Oof, how could I forget that. I will try that.

**Reply by LautiLosio** [reply]
~~This worked perfectly.~~ Thank you SO much! You just made my week.

I found a _minor_ issue: the app crashes when I try to make a new note...

**Reply by LautiLosio** [reply]
It was an issue with the version of the app.
4.3.7 -> Not working
3.7.5m -> Works perfectly

**Reply by reiryuki** [reply]
https://t.me/androidryukimodsdiscussions/2618

**Reply by 123Sanyam123** [reply]
hello. Were you able to sync the notes over mi cloud?
